modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'will_paginate/array'
+
+require_dependency 'va_facilities/application_controller'
+require_dependency 'va_facilities/pagination_headers'
+require_dependency 'va_facilities/geo_serializer'
+require_dependency 'va_facilities/csv_serializer'
+
+module VaFacilities
+  module V1
+    class NearbyController < ApplicationController
+      include ActionController::MimeResponds
+      include VaFacilities::PaginationHeaders
+      skip_before_action(:authenticate)
+      before_action :set_default_format
+      before_action :validate_params, only: [:index]
+
+      REQUIRED_PARAMS = %i[street_address city state zip].freeze
```

Comment:
Shall we use strong parameters instead?
https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters